.github/workflows/fetch-bank-holidays.yml

```diff
@@ -0,0 +1,41 @@
+name: Fetch bank holidays JSON from the gov.uk API
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
Is it worth setting this (at least temporarily) to `'0 8 * * *'` so it'll next run at 9am (our time)?

Reply:
good idea